static/scripts/administration.js

```diff
@@ -15,10 +15,17 @@ window.addEventListener('DOMContentLoaded', () => {
 	if (filterModule) {
 		filterModule.addEventListener('newFilter', (e) => {
 			const filter = e.detail;
+
+			const page = getQueryParameterByName('p');
+			const showTab = getQueryParameterByName('showTab');
+
 			const newurl = `?filterQuery=${
 				escape(JSON.stringify(filter[0]))
-			}&p=${
-				getQueryParameterByName('p')}`;
+			}${
+				page ? `&p=${page}` : ''
+			}${
+				showTab ? `&showTab=${showTab}` : ''
+			}`;
```

Comment:
let page = getQueryParameterByName('p');
page = page ? `&p=${page}` : '';

let showTab = getQueryParameterByName('showTab');
showTab = showTab ? `&showTab=${showTab}` : '';

const filterQuery =`?filterQuery=${escape(JSON.stringify(filter[0]))};

const newurl = `${ filterQuery }${ page }${ showTab  };

Reply:
You're absolutely right. This makes the code more readable. Thank you for the input.